scripts/install_karavi_auth.sh

```diff
@@ -91,4 +122,20 @@ else
 fi
 
 sh ./policies/policy-install.sh
+
+
+if [ $STATIC_PORT -eq 1 ]
+then
+  while [ $($K3S kubectl get svc -n kube-system | grep traefik | wc -l) -ne 1 ]
+  do
+        echo "Waiting for traefik service to be available ..."
+        sleep 10s
+  done
+
+  if [ $($K3S kubectl get svc -n kube-system | grep traefik | wc -l) -eq 1 ]
+  then
+        sh ./traefik_nodeport.sh --traefik_web_port $webPort --traefik_websecure_port $websecurePort
```

Comment:
Is there a need to have a separate script? Since the inputs for this configuration are already defined in this script , may just want to call a function here.

Reply:
@sharmilarama, the intent here is to give more flexibility to customers. In case customers forget to set the Nodeports during installation/upgrade, they can do it in a later stage using traefik_nodeport.sh. Also, it gives the possibility to change the Nodeports in a later stage without having to touch any other configuration.